src/main/java/de/muenchen/dave/services/messstelle/MesswerteService.java

```diff
@@ -77,8 +79,15 @@ public LadeProcessedMesswerteDTO ladeMesswerte(final String messstelleId, final
         if (CollectionUtils.isNotEmpty(intervals)) {
             processedZaehldaten.setTagesTyp(TagesTyp.getByIntervallTyp(intervals.getFirst().getTagesTyp()));
         }
-        processedZaehldaten.setRequestedMeasuringDays(ChronoUnit.DAYS.between(options.getZeitraum().getFirst(), options.getZeitraum().getLast()) + 1);
-        processedZaehldaten.setIncludedMeasuringDays(response.getIncludedMeasuringDays());
+
```

Comment:
Hier in einen Kommentar den Grund beschreiben weshalb die Logik angewendet wird.